integration_tests/integration/createLicence.spec.js

```diff
@@ -0,0 +1,15 @@
+const IndexPage = require('../pages/index')
+
+context('Create a Licence', () => {
```

Comment:
Should there be an Axe test here to verify that the page is accessible? Or better to put these accessibility checks in a different test?

Reply:
Added Axe testing to the verifyPage function - this allows accessibility to be tested implicitly each time a page is loaded during the test